.circleci/config.yml

```diff
@@ -33,10 +33,10 @@ jobs:
     steps:
       - attach_workspace:
           at: ~/project
-      - run: 
+      - run:
           command: |
-            npm install publish
-            ./node_modules/.bin/publish
+            npm install -g publish
```

Comment:
I wonder if we could just use `npx publish`?

Reply:
It's definitely worth a try